app/views/pupils/schools/_explore_old_charts.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="row dashbards">
+  <% if school.meters_for_supply(:electricity).any? %>
```

Comment:
Use the school config

Reply:
Maybe we should remove all access to these methods from the school?

Reply:
Yep